apisix/cli/ngx_tpl.lua

```diff
@@ -57,7 +57,7 @@ env {*name*};
 {% end %}
 {% end %}
 
-{% if stream_proxy then %}
+{% if stream then %}
```

Comment:
Because it said before that the default is on, so the stream will be loaded by default, and when the user turns on the stream_proxy, then turn on the relevant content

Reply:
After this change, the stream module is turned on by default, which is not useless in major cases.

Reply:
Indeed, if the stream function is not used, it is redundant. Then revert to this?
```
stream_proxy: # TCP/UDP proxy
  tcp: # TCP proxy address list
    - 9100
    - "127.0.0.1:9101"
  udp: # UDP proxy address list
    - 9200
    - "127.0.0.1:9211"
  lua_shared_dict:
    lrucache_lock_stream: 10m
```
Is there a good way to enable it by default in stream_proxy?

Reply:
Please revert this change.

Reply:
OK, then I will change it back